codalab/apps/api/views/competition_views.py

```diff
@@ -693,10 +693,35 @@ class SubmissionScoreView(views.APIView):
     """
     Provides a way to grab scores given a specific PK and the owner is the one making the request
     """
+
+    logs_to_grab = [
+        'inputfile',
+        'output_file',
+        'private_output_file',
+        'stdout_file',
+        'stderr_file',
+        'history_file',
+        'scores_file',
+        'detailed_results_file',
+        'prediction_runfile',
+        'prediction_output_file',
+        'exception_details',
+        'prediction_stdout_file',
+        'prediction_stderr_file',
+        'ingestion_program_stdout_file',
+        'ingestion_program_stderr_file',
+    ]
+
     def get(self, request, *args, **kwargs):
         submission_id = self.kwargs.get('submission_id')
         try:
             sub = CompetitionSubmission.objects.get(pk=submission_id)
+            log_sas_urls = {}
+            if sub:
+                for log_attr in self.logs_to_grab:
+                    # TODO: Will this cause errors when None? Did not have this occur when testing with Eric
+                    if hasattr(getattr(sub, log_attr), 'file'):
+                        log_sas_urls[log_attr] = _make_url_sassy(getattr(sub, log_attr).file.name, permission='r', duration=60 * 60 * 24)
```

Comment:
why not make expiration like... 10 years?

Reply:
I sent you a message on this. I thought it was just closer to our best practices, especially considering security.

Reply:
👍